spark/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -562,6 +562,119 @@ class DeltaTable private[tables](
           TableFeatureProtocolUtils.FEATURE_PROP_SUPPORTED))
     toDataset(sparkSession, alterTableCmd)
   }
+
+  /**
+   * Create a shallow clone of this DeltaTable to a new location.
+   *
+   * @param target The path or table name to create the clone
+   * @since 4.0.0
+   */
+  def clone(target: String): DeltaTable = {
+    clone(target, false)
```

Comment:
Could we call `executeClone(..., ...)` to be coherent with the other `DeltaTable` APIs?

Reply:
Other uses aren't really the same as what we're doing here, which is default values vs optional values. We would have to hard code the same default values in multiple places which seems like an anti-pattern

Reply:
I think we should still order the functions the other way around though, when we look at the code we often look from top to bottom

Reply:
Reordered